src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -354,7 +360,8 @@ private static void createSourceBQTableWithQueries(String bqCreateTableQueryFile
       // Iterator on TableResult values in getSoleQueryResult method throws NoSuchElementException
     }
     PluginPropertyUtils.addPluginProp("bqSourceTable", bqSourceTable);
-    BeforeActions.scenario.write("BQ Source Table " + bqSourceTable + " created successfully");
+    System.out.println(bqSourceTable);
+//    BeforeActions.scenario.write("BQ Source Table " + bqSourceTable + " created successfully");
```

Comment:
Remove the commented lines.

Reply:
done
commit id- https://github.com/cloudsufi/google-cloud/commit/defc54df110255ef539dea8d206481cc145b2fec